### PR TITLE
open-cluster-management/backlog#1704 Add bm asset

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: manage_cluster/bare_assets.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+You need the following prerequisites before creating a bare metal asset:
+
+* A deployed Red Hat Advanced Cluster Management for Kubernetes hub cluster
+
+* Access for your Red Hat Advanced Cluster Management for Kubernetes hub cluster to connect to the bare metal asset.
+
+* A configured bare metal asset, and log in credentials with the required permissions to log in and manage it.
+
+* Login credentials for your bare metal asset, which include the following items for the asset that are provided by your administrator:
```
Comment (dockerymick) [inline manage_cluster/bare_assets.md:21]: I would update this to the following:
(2 spaces before the note)
**Note**: Login credentials for your bare metal asset include the following items for the asset, which are provided by your administrator:

File: manage_cluster/bare_assets.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  * User name
+  * Password
+  * Baseboard Management Controller Address
```
Comment (dockerymick) [inline manage_cluster/bare_assets.md:24]: Should all of the words be capitalized? Is there an acronym?

File: manage_cluster/bare_assets.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  * Boot NIC MAC address
+
+## Creating your bare metal asset with the Red Hat Advanced Cluster Management for Kubernetes console
+
+To create a bare metal asset using the Red Hat Advanced Cluster Management for Kubernetes console, complete the following steps: 
+
+1. From the navigation menu, navigate to **Automate infrastructure** > **Bare metal assets**.
+
+2. On the Bare metal assets page, Click **Create bare metal asset**.
```
Comment (dockerymick) [inline manage_cluster/bare_assets.md:33]: I'm not sure if we should italicize page names. We should probably rediscuss this as a team for consistency throughout our repo.

File: manage_cluster/bare_assets.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+3. Enter a name for your asset. This name is used to identify this asset when creating a cluster.
```
Comment (dockerymick) [inline manage_cluster/bare_assets.md:35]: I think this is a bit wordy, I suggest: "Enter a name for your asset when you create a cluster. "

File: manage_cluster/bare_assets.md (hunk 3 above)
Comment (dockerymick) [reply manage_cluster/bare_assets.md:33]: Confirmed that we italicize page names. Update Bare metal assets page to _Bare metal assets_ page

File: manage_cluster/bare_assets.md (hunk 1 above)
Comment (cadawson) [reply manage_cluster/bare_assets.md:21]: Good idea. Let's try that.

File: manage_cluster/bare_assets.md (hunk 4 above)
Comment (cadawson) [reply manage_cluster/bare_assets.md]: Rewrote it to make it shorter.

File: manage_cluster/bare_assets.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+2. On the *Bare metal assets* page, Click **Create bare metal asset**.
+
+3. Enter a name for your asset that identifies it when you create a cluster. 
+
+  **Tip:** You can view the `yaml` content updates as you enter the information in the console by setting the *YAML* switch to **ON**. 
+
+4. Enter the namespace for your bare metal asset. This namespace must be the same as the namespace where you create your cluster that uses this asset.
+
+5. Enter the Baseboard Management Conroller address. This is the controller that enables communication with the host. The following protocols are supported:
```
Comment (cdoan1) [inline manage_cluster/bare_assets.md:40]: Do we have some real examples of the protocols?

```
ipmi://192.168.100.12
redfish://192.168.100.14
```

File: manage_cluster/bare_assets.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+1. From the navigation menu, navigate to **Automate infrastructure** > **Bare metal assets**.
+
+2. On the *Bare metal assets* page, Click **Create bare metal asset**.
+
+3. Enter a name for your asset that identifies it when you create a cluster. 
+
+  **Tip:** You can view the `yaml` content updates as you enter the information in the console by setting the *YAML* switch to **ON**. 
+
+4. Enter the namespace for your bare metal asset. This namespace must be the same as the namespace where you create your cluster that uses this asset.
```
Comment (cdoan1) [inline manage_cluster/bare_assets.md:38]: Are we sure that the namespace of the asset should be the namespace of the future cluster ?

The BMA records are added to a namespace. As long as the user, has access to the namespace, he/she can associate them to to the provisioning manifests later, and in order to do this.

So, I would change this to:

4. Enter the namespace where the bare metal assets will be created. Users with access to this namespace will be able to associate these assets during the provisioning step.

Comment (cdoan1) [reply manage_cluster/bare_assets.md:38]: The current order which we test is:

1. create BMA
2. create Provider Connection
3. provision BM
4. import BM into ACM/OCM, which creates the namespace in ACM/OCM

File: manage_cluster/bare_assets.md (hunk 5 above)
Comment (cadawson) [reply manage_cluster/bare_assets.md:40]: I am not sure what you mean by real examples, but I added some links where people can get more information about the specs.  Will that work?

File: manage_cluster/bare_assets.md (hunk 6 above)
Comment (cadawson) [reply manage_cluster/bare_assets.md]: Made this change.
